### PR TITLE
Use example syncer tags instead of hard-coded examples in doc

### LINKED ISSUE #11603: Use example syncer tags instead of hard-coded examples in doc
After #11499 is merged, use the following munge example syncer tag to replace examples (yaml/json) in .md files 
```
<!-- BEGIN MUNGE: EXAMPLE pod.yaml -->
<!-- END MUNGE: EXAMPLE -->
```

#11447 

cc @bgrant0607

### PR DESCRIPTION
**Pull request by janetkuo**
Fixes #11603 (based on #11499, so only need to review the 2nd commit)

cc @bgrant0607

### CODE REVIEW

**Comment by lavalamp** [comment]
Hm, it looks like a lot of the examples have diffs. We need to figure out in each case if the docs or the file was better.

File: docs/user-guide/downward-api.md [hunk 1]
```diff
@@ -91,17 +93,20 @@ spec:
       image: gcr.io/google_containers/busybox
       command: [ "/bin/sh", "-c", "env" ]
       env:
-        - name: MY_POD_NAME
+        - name: POD_NAME
```
Comment (bgrant0607) [inline docs/user-guide/downward-api.md]: Please use the MY_ names

File: docs/user-guide/simple-yaml.md [hunk 2]
```diff
@@ -47,13 +47,15 @@ $ kubectl create -f ./pod.yaml
...
+<!-- BEGIN MUNGE: EXAMPLE pod.yaml -->
+
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: nginx
   labels:
-    app: nginx
+    name: nginx
```
Comment (bgrant0607) [inline docs/user-guide/simple-yaml.md]: Please use app, not name

File: docs/user-guide/walkthrough/k8s201.md [hunk 3]
```diff
@@ -115,26 +122,29 @@ metadata:
...
+    name: nginx
+  # template defines the 'cookie cutter' used for creating
   # new pods when necessary
   template:
     metadata:
       labels:
         # Important: these labels need to match the selector above
         # The api server enforces this constraint.
-        app: nginx
+        name: nginx
```
Comment (bgrant0607) [inline docs/user-guide/walkthrough/k8s201.md]: app not name, here and everywhere.

File: docs/user-guide/walkthrough/k8s201.md [hunk 4]
```diff
@@ -171,18 +183,21 @@ metadata:
...
+      # the container on each pod to connect to, can be a name
+      # (e.g. 'www') or a number (e.g. 80)
+      targetPort: 80
+      protocol: TCP
   # just like the selector in the replication controller,
   # but this time it identifies the set of pods to load balance
   # traffic to.
   selector:
-    app: nginx
+    name: nginx
```
Comment (bgrant0607) [inline docs/user-guide/walkthrough/k8s201.md]: app

**Comment by bgrant0607** [comment]
I don't have time for a thorough review at the moment. Let's please not merge this yet.

File: docs/getting-started-guides/logging.md [hunk 5]
```diff
@@ -163,19 +169,31 @@ metadata:
...
-    image: gcr.io/google_containers/fluentd-gcp:1.6
+    image: gcr.io/google_containers/fluentd-gcp:1.8
+    resources:
+      limits:
+        cpu: 100m
+        memory: 200Mi
     env:
-    - name: FLUENTD_ARGS
-      value: -qq
+    - name: "FLUENTD_ARGS"
```
Comment (lavalamp) [inline docs/getting-started-guides/logging.md]: these quotes should be unnecessary.

File: docs/user-guide/walkthrough/README.md [hunk 6]
```diff
@@ -159,10 +161,13 @@ spec:
     - name: redis-persistent-storage
       mountPath: /data/redis
   volumes:
-  - name: redis-persistent-storage
-    emptyDir: {}
+    - name: redis-persistent-storage
```
Comment (lavalamp) [inline docs/user-guide/walkthrough/README.md]: I think the old indentation is correct?

File: docs/user-guide/walkthrough/k8s201.md [hunk 7]
```diff
@@ -115,26 +122,29 @@ metadata:
...
         # The api server enforces this constraint.
-        app: nginx
+        name: nginx
     spec:
       containers:
-      - name: nginx
-        image: nginx
-        ports:
-        - containerPort: 80
+        - name: nginx
```
Comment (lavalamp) [inline docs/user-guide/walkthrough/k8s201.md]: old indentation. some other places, too.

Comment (lavalamp) [inline docs/user-guide/walkthrough/k8s201.md]: might be a spaces vs tabs problem, I don't know if our yaml files are standardized.

File: docs/user-guide/downward-api.md (hunk 1 above)
Comment (JanetKuo) [inline docs/user-guide/downward-api.md]: Fixed

File: docs/user-guide/simple-yaml.md (hunk 2 above)
Comment (JanetKuo) [inline docs/user-guide/simple-yaml.md]: Fixed

File: docs/user-guide/walkthrough/k8s201.md (hunk 3 above)
Comment (JanetKuo) [inline docs/user-guide/walkthrough/k8s201.md]: Fixed

File: docs/user-guide/walkthrough/k8s201.md (hunk 4 above)
Comment (JanetKuo) [inline docs/user-guide/walkthrough/k8s201.md]: Fixed

File: docs/getting-started-guides/logging.md (hunk 5 above)
Comment (JanetKuo) [inline docs/getting-started-guides/logging.md]: Fixed

File: docs/user-guide/walkthrough/README.md (hunk 6 above)
Comment (JanetKuo) [inline docs/user-guide/walkthrough/README.md]: Fixed

File: docs/user-guide/walkthrough/k8s201.md (hunk 7 above)
Comment (JanetKuo) [inline docs/user-guide/walkthrough/k8s201.md]: Fixed

**Comment by JanetKuo** [comment]
Updated yaml/json files to fix the indentation and label errors.

**Comment by bgrant0607** [comment]
Thanks. I'm fine with all the example config changes. Hold removed.

@mikedanese At some point we should update the examples to adhere to best practices.

**Comment by lavalamp** [comment]
If Brian's happy, I'm happy. Can you rebase, please?

**Comment by JanetKuo** [comment]
Rebased. Thanks!

**Comment by JanetKuo** [comment]
It's now ready for merge.

**Pull request merged**